infra/buckets/gs.tf

```diff
@@ -41,7 +41,7 @@ resource google_storage_bucket dss_gs_checkout_bucket {
     }
     condition {
       age = "${var.DSS_BLOB_TTL_DAYS}"
-      matches_storage_class = ["DURABLE_REDUCED_AVAILABILITY"]
+      matches_storage_class = ["STANDARD"]
```

Comment:
can we change this for the test buckets too?

Reply:
The test bucket lifecycle rule is not based on storage class, so it doesn't need to be updated.